packages/devtools_server/lib/src/server.dart

```diff
@@ -113,6 +115,18 @@ Future<HttpServer?> _serveDevToolsWithArgs(
     return null;
   }
 
+  if (version) {
+    final versionStr = await _getVersion();
+    printOutput(
+      'Dart DevTools version $versionStr',
+      {
+        'version': versionStr,
```

Comment:
It looks like `_getVersion` can return the string `'unknown'` if it can't get the version. Would it be better to do something else (like `null` or `0.0.0`) for the JSON mode?

Reply:
Just returning `'unknown'` is probably more clear than returning `0.0.0` or `null`, so I think this makes the most sense for now.